src/main/java/jenkins/branch/buildstrategies/basic/AllBranchBuildStrategyImpl.java

```diff
@@ -77,7 +78,8 @@ public boolean isAutomaticBuild(@NonNull SCMSource source, @NonNull SCMHead head
                 source,
                 head,
                 currRevision,
-                prevRevision
+                prevRevision,
+                taskListener
```

Comment:
this is a problem...this strategy wraps other strategies from previous versions that might not yet have implemented this new interface with the extra method...as such it'll cause abstract method problem.

Reply:
i think you'd need to add a TaskAwareBuildStrategy interface that has this new method and check for instanceof that.  maybe there's a more common approach in jenins, but that's how i've done it in java in the past.

Reply:
Hi @drdamour I am fixing this in #5 - once the build is successful... would you mind to give a try, please?

Reply:
#5 won't fix this..in fact it makes it even worse as you have further diverged from the existing interface!

Reply:
https://issues.jenkins-ci.org/browse/JENKINS-56984